### ISSUE TITLE
Proofs

### ISSUE THREAD

**Issue by nadalin** [w3c/vc-data-model#611]
"For a credential or presentation to be verifiable, at least one proof mechanism MUST be expressed." I assume proofs are optional, since it is not stated that they are mandatory and this statement just says if you want the credential to be verifiable it MUST have at least 1 proof mechanism. This needs better wording to state what is mandatory and what is optional

**Comment by TallTed** [comment]
Perhaps 
```
At least one proof mechanism, and the details necessary to evaluate that 
proof, MUST be expressed for a <a>credential</a> or <a>presentation</a> 
to be <a>verifiable credential</a> or <a>verifiable presentation</a>.
```

**Comment by nadalin** [comment]
Better

**Comment by burnburn** [comment]
I believe this has been completed.  Marking with 7 Day close

**Comment by burnburn** [comment]
WG resolution on 28 May 2019 call:

RESOLUTION: Issue #611 is addressed via PR #617, which makes non-normative clarifications to the specification regarding proof details and mechanism. Issue #611 will be closed immediately since the 7 day wait period has expired.
